pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```diff
@@ -8,7 +8,7 @@
 , nodejs
 , fetchYarnDeps
 , fixup_yarn_lock
-, electron_24
+, electron_27
```

Comment:
Perhaps simply accept `electron` as an argument and set the version in `all-packages.nix` as done in e.g. Element Desktop [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/element/element-desktop.nix#L11) and [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L5210)? It would mean the below changes are unnecessary and also make version overrides easier/clearer in future.

Reply:
Sure, done.

Reply:
So it was only held back to `electron_24` instead of always using latest because of newer versions being unable to screenshare

If that's fixed I would actually be inclined to always use latest until it becomes a problem again rather than hardcoding `electron_27` anywhere

Thoughts?

Reply:
Following the upstream supported version unless there are significant reasons not to do so seems the most logical to me, but my opinion is not that strong. I haven't tested screen sharing with the electron 27 version, but given the upstream support I would assume it works?

The main reason for this PR is that `electron_24` has been marked as insecure on `nixos-unstable`.

Reply:
I'll go ahead and merge this for now, but I'll open a new PR to discuss in the next few days